src/main/java/com/hedvig/backoffice/graphql/resolvers/QuoteSchemaResolver.kt

```diff
@@ -0,0 +1,13 @@
+package com.hedvig.backoffice.graphql.resolvers
+
+import com.coxautodev.graphql.tools.GraphQLResolver
+import com.hedvig.backoffice.graphql.types.Quote
+import com.hedvig.backoffice.services.underwriter.UnderwriterService
+
+class QuoteSchemaResolver(
```

Comment:
Resolvers are on "the parent" not what they resolve afaik
```suggestion
class QuoteResolver(
```